src/cmd/facette/service.go

```diff
@@ -33,10 +33,28 @@ func NewService(config *config) *Service {
 
 // Run starts the service processing.
 func (s *Service) Run() error {
-	var err error
+	var (
+		loggers = make([]interface{}, 0)
```

Comment:
Looking at the code I'm under the impression that we can use file-based, file-based and syslog-based, but not syslog-based only logging? Did I miss something?

Reply:
The impression you're under is correct. Do we want users to be able to completely disable file logging, even if unconfigured/configured with default it only prints info-level messages to `stderr`?

Reply:
To my opinion, there could be use cases implying to only send logs to syslog without having to keep an additional local log file (or `stderr` output).

Reply:
OK, then how about if `config.LogPath == ""` (the default) we don't enable file logging (and enable logging to stderr if `config.LogPath == "-"`?

Reply:
We can surely disable file logging if the `LogPath` is empty.

But in this case, it could be useful to stick to the old behavior by setting the default value to `-`. WDYT?

Reply:
LGTM, will do.